app/mailers/concerns/notify_template_methods.rb

```diff
@@ -15,4 +15,8 @@ def support_email_address
   def safe_nil(value)
     value || ''
   end
+
+  def url_expiry_date
+    (Date.today + 7.days).strftime('%d %B %Y')
```

Comment:
Do we know if the GDS style guide specifies zero padding or not. For some reason I thought it is against zero padding as when months are numeric they appear as `3` not `03`. As per the date hint text on this page https://design-system.service.gov.uk/patterns/dates/. If so, then this needs to be `%-d`

Reply:
Yeah it looks like it does in this example. Notice it is `4` and not `04` https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style 

- Dates
- use upper case for months: January, February
- do not use a comma between the month and year: 4 June 2017

Reply:
Updated thanks, good catch